src/classes/STG_PanelContactRoles_CTRL.cls

```diff
@@ -194,4 +194,94 @@ public with sharing class STG_PanelContactRoles_CTRL extends STG_Panel {
         }
         return results;
     }
+
+    /**
+    * @description The current setting being stored on the Opportunity Contact Role Default Role
+    */
+    public String individualOppsContactRole {
```

Comment:
Perhaps var names can be changed a bit so it is clear if it is a field name, field label, field picklist value or the field picklist label. Since the `individualOppsContactRole` and other similar vars below are value *labels* of the custom settings field representing OpportunityContactRole Role picklist field value(s), can we somehow rename them? For example:
- `individualOppsContactRole` -> `selectedOptionDefaultContactRole`
- `organizationOppsContactRole` -> `selectedOptionOrgDefaultContactRole`
- `matchedDonorRole` -> `selectedOptionMatchedDonorRole` 
- `hhMemberContactRole` -> `selectedOptionHHMemberContactRole`
- `honoreeOppContactRole` -> `selectedOptionHonoreeContactRole`
- `recipientOppContactRole` -> `selectedOptionNotficationRecipientRole`?